third_party/move/move-vm/types/src/values/values_impl.rs

```diff
@@ -505,6 +513,13 @@ impl ContainerRef {
     }
 }
 
+#[cfg(test)]
+impl Value {
+    pub fn copy_value_with_depth(&self, max_depth: u64) -> PartialVMResult<Self> {
+        Ok(Self(self.0.copy_value(1, Some(max_depth))?))
```

Comment:
Shouldn't there also be a negative test?

Reply:
I have `value_depth_tests::test_copy_value` which also tests for +-1 around boundary? Which negative test do you mean here?